app/src/test/java/com/github/multimatum_team/multimatum/FirebaseGroupRepositoryTest.kt

```diff
@@ -98,6 +98,19 @@ class FirebaseGroupRepositoryTest {
         )
     }
 
+    /* Untestable even with mocking
+    @Test
+    fun `Removing a group member removes it from the database`() = runTest {
```

Comment:
Because `removeMember` is implemented as
```kt
        groupsRef
            .document(groupID)
            .update("members", FieldValue.arrayRemove(memberID))
            .await()
```
But to mock this, I need to do
```
        `when`(document.update(eq("members"), any())).then {
            val fieldValue = it.getArgument<FieldValue>(1)
            ...
        }
```
But I'm blocked here, there is no way I can recover `memberID` from `FieldValue` (no public getter is made available) by the Firebase API. Another solution I tried was:

```kt
        `when`(document.update(eq("members"), FieldValue.arrayRemove(anyString()))).then {
            ...
        }
```
but mockito does not support nested matchers.